Makefile

```
@@ -1,65 +1,28 @@
 .PHONY: test
 test:
-ifeq ($(IS_IN_CONTAINER),0)
-	go generate ./app && \
-	go test -v --cover ./app
-else
-	make devcontainer-up && devcontainer exec --workspace-folder ./ make test
-endif
+	go test -v --cover ./...
 
 .PHONY: e2e-test
+e2e-test: PORT ?= 8080
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)